tests/unit/moduleapi/reply.tcl

```diff
@@ -28,6 +28,29 @@ start_server {tags {"modules"}} {
             assert_equal 3.141 [r rw.double 3.141]
         }
 
+        test "RESP$proto: RM_ReplyWithDouble: inf" {
+            if {$proto == 2} {
+                assert_equal "inf" [r rw.double inf]
+                assert_equal "-inf" [r rw.double -inf]
+            } else {
+                assert_equal Inf [r rw.double inf]
+                assert_equal -Inf [r rw.double -inf]
```

Comment:
https://github.com/redis/redis/actions/runs/3457509357/jobs/5770995313#step:6:906
look like this one is not the same on mac
```
*** [err]: RESP3: RM_ReplyWithDouble: inf in tests/unit/moduleapi/reply.tcl
Expected 'Inf' to be equal to 'inf' (context: type eval line 6 cmd {assert_equal Inf [r rw.double inf]} proc ::test)
```